doc/Cellpose-StarDist-Segmentation.ipynb

```diff
@@ -0,0 +1,454 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cellpose/StarDist Segmentation\n",
+    "\n",
+    "In this notebook, we will segment HeLa cell cytoplasms and nuclei with Cellpose and StarDist respectively. Finally we will  convert the resulting index images into ImgLib2 and legacy ImageJ regions of interest (ROI). This notebook requires Cellpose and StarDist in your conda/mamba environment. You can use the following `environment.yml` file below to create an environment with Cellpose, StarDist and PyImageJ.\n",
```

Comment:
When I wanted to write about this new feature I at first was just going to write a markdown page because, in my opinion, most users will read our documentation via our readthedocs site. Additionally this notebook requires external packages that are note PyImageJ dependencies (_i.e._ `stardist` and `cellpose`). Because of how we build our docs (we don't actually run the notebooks on the readthedocs site, instead we just push the contents and it gets built) it decided to strike a balance and write the document kind of like a page to read (if you were none the wiser) but its also a notebook that you can pull and given the correct environment, give it a run. Perhaps this has lead to a confusing notebook?

My overall goal is really to show case this feature, which is why it's part of a use case and not the tutorial notebooks. I'm not really sure how many users are _actually_ pulling the notebooks and running them as opposed to reading them online and taking the chunks they need for whatever project they're working with. I'm also hesitant to ship the environment file with the PyImageJ docs as we already have environment files on the root which might confuse someone. Thoughts?

Reply:
Notebooks can dynamically install additional dependencies using conda, pip, or both, via the bang `!` syntax. Maybe this would be a good way forward to ensure this notebook actually works using the existing `environment.yml` in the PyImageJ base as a starting point?

Reply:
I like this idea. Let me give that a try.

Reply:
Using the `!` bang syntax worked (once I figured out which packages need to come from where) on Linux, macOS and Windows. :+1: .